components/command-arguments.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <ul class="arg-list">
+    <li v-for="(arg, index) in command.args" :key="`arg-${index}`" class="arg">
+      <p class="arg__name">
+        {{ arg.name }} <span>{{ '('+(arg.required?'required':'optional')+')' }}</span>
```

Comment:
Why not use a template literal like on line 3?

Reply:
I must have missed that one! Well noticed, thanks